MobiFlight/MobiFlightLedModule.cs

```diff
@@ -89,6 +98,40 @@ public void Display(int subModule, String value, byte points, byte mask)
             CmdMessenger.SendCommand(command);
         }
 
+        static public byte CalculateCorrectPoints(string value, byte mask)
+        {
+            byte points = 0;
+
+            // we start with the last character in the value string
+            int positionInValue = value.Length-1;
+
+            // we go over all 8 potential digits
+            for (byte digit=0; digit<8; digit++)
+            {
+                // if the digit is not active, go to the next
+                if (((1 << digit) & mask) == 0)
+                    continue;
+
+                // stop when you ran out of value to display
+                if (positionInValue < 0)
+                    break;
+
+                // when we have a decimal point at the current position
+                if (value[positionInValue]=='.')
+                {
+                    // activate the point at the current digit
+                    points |= (byte)(1 << digit);
+                    
+                    // then we stay on the digit one more time
```

Comment:
I don't understand this code comment. Why does it say "stay on the digit" but decrement `digit`?